contracts/HeapOrdering.sol

```diff
@@ -244,12 +244,15 @@ library HeapOrdering {
 
         // Swap the last account and the account to remove, then pop it.
         swap(_heap, index, accountsLength - 1);
-        if (_size == accountsLength) _heap.size = _size - 1;
+        if (_size == accountsLength) {
+            _size--;
+            _heap.size = _size;
```

Comment:
and if you try just: `_heap.size = _size--;`?

And in an unchecked?

Reply:
Yes it has already been changed ;-)

Reply:
wut? I'm not sure what happened 😅

Reply:
I think that you started a review then I changed it and then you submitted your review :-)